crypto_sdk/src/contracts/tests/ERC721TestMintable.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
what is this file about?

Reply:
> what is this file about?

it's test implementation of ERC721 with mint function